cryptohelp/ssl.py

```diff
@@ -112,12 +107,16 @@ def create_csr(
             )
     )
 
-    if _dns_names:
+    if dns_names:
+        _dns_names = [x509.DNSName(n) for n in dns_names]
```

Comment:
what does x509.DNSName return? Is it possible for this to return an empty list, hence the additional check we had previously?

What testing did you do? I don't see it in this comment: https://github.com/ccxtechnologies/builder/issues/995#issuecomment-1766501188

Same thing for x509.IPAddress

Reply:
I've done simple testing with ipython and also loaded code to vm_server, but I can look more into the source code of the x509.DnsName() method to confirm that it's not possible to return an empty list.
Thanks!

Reply:
I've added testing to the end of this comment: https://github.com/ccxtechnologies/builder/issues/995#issuecomment-1766501188